specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.2/keys.json

```
@@ -1542,17 +1541,17 @@
         "certificates": {
           "type": "string",
           "format": "base64url",
-          "description": "The certificates used for validating attestation."
+          "description": "A base64url encoded string containing certificates in PEM format, used for attestation validation."
```

```suggestion
          "description": "A base64url-encoded string containing certificates in PEM format, used for attestation validation."
```

But why are you having them encode a PEM file? It's already encoded. And your other APIs don't encode PEM. That said, whether you make them double encode it or not, what line endings should they use? Or do you support all 3 (CR, LF, and CRLF) on the backend? That's not true (or wasn't, for a time) of KV.

---

This PEM file is coming from the HSM api layer. This HSM api layer returns bytes, Service side, in our platform transform layer, we are transforming these bytes into Base64 encoding string similar to other large strings. The PEM format certificate has LF line endings. One quick point is that this certificate bundle is only one way, in that the certificate bundle is downloaded by customers after invoking this endpoint and the certificate bundle is never uploaded to the service side.